### PR TITLE
DOC Fix bad syntax for enum.

### PR DESCRIPTION
**Pull request by maxime-rainville**
## Parent issue
- https://github.com/silverstripe/developer-docs/issues/288

### CODE REVIEW

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Please target the `5.0` branch so that it can appear on the docs site as soon as it's merged rather than waiting for the next recipe minor release.

**Comment by maxime-rainville** [comment]
Retargetted.

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**
